operator/pkg/ciliumendpointslice/manager.go

```diff
@@ -453,11 +464,10 @@ func (c *cesMgr) clearRemovedCEPs(cesName string, remCEPs map[string]struct{}) {
 	// Check if CES exists in local cache
 	if ces, ok = c.desiredCESs.getCESTracker(cesName); !ok {
 		log.WithFields(logrus.Fields{
-			logfields.CESName: cesName,
+			"CESName": cesName,
```

Comment:
Why is this switching from the string variable to open-writing the name? The reason we use `logfields.CESName` for cases like this is to ensure that all logs that have a field where the field is the CES name will all have the exact same logging key in the structured logs.

Please switch this and all of the other cases to `logfields.CESName`.

Reply:
Thanks for review..
Had updated.
But i still found the log label have manual-define, like 'CEPCount', 'totalCEPCount'..
Is that ok? 
or we should move it to the "pkg/logging/logfields/logfields.go" to define it?

Reply:
Could you define all of the keys in `pkg/logging/logfields/logfields.go`? This will keep them all consistent across the codebase. Thanks.

Reply:
yes, updated.